rockcraft.yaml

```diff
@@ -73,11 +73,20 @@ parts:
     build-snaps:
       - go/1.21/stable
     build-environment:
-      - CGO_ENABLED: 0
+      - CGO_ENABLED: "0"
```

Comment:
Just a **NONBLOCKING** question (curious to learn): why we specifically convert it to string?

Reply:
Reason is stupid: Vscode was complaining that this must be a string (see [rockcraft.json schema](https://raw.githubusercontent.com/canonical/rockcraft/main/schema/rockcraft.json))